test/cmd/deploy_prometheus.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
+trap os::test::junit::reconcile_output EXIT
+
+os::test::junit::declare_suite_start "cmd/create"
+
+os::cmd::expect_success 'make build'
+
+TEST_DIR=$(pwd)
+
+os::cmd::expect_success 'oc new-project prom'
+
+os::cmd::expect_success 'oc create -f "$TEST_DIR"/spark-metrics-template.yaml'
+
+os::cmd::expect_success 'oc new-app --template=spark -p MASTER_NAME=master -p WORKER_NAME=worker -p SPARK_METRICS_ON=prometheus'
+
+os::cmd::try_until_text 'oc get pods' 'worker'
```

Comment:
Ditto the deploy_jolokia.sh comment, this can be hyper-focused on enabling the prometheus endpoint.

Reply:
changed